packages/manager/apps/pci-rancher/src/components/layout-helpers/CreateRancher/CreateRancher.tsx

```diff
@@ -137,7 +137,19 @@ const CreateRancher: React.FC<CreateRancherProps> = ({
     }
 
     if (selectedVersion === null && versions?.length) {
-      setSelectedVersion(versions?.filter((v) => v.status === 'AVAILABLE')[0]);
+      const availableVersions = versions.filter(
+        (version) => version.status === 'AVAILABLE',
+      );
```

Comment:
**nitpick:**  Looks like only first match is used. So `Array.prototype.find` is the right function to use.
```suggestion
      const availableVersions = versions.find(
        ({ status }) => status === 'AVAILABLE',
      );
```